src/koreo_tooling/langserver/codelens.py

```diff
@@ -397,9 +397,14 @@ def _code_lens_inputs_action(test_name: str, test_result: TestResults):
     )
 
     indent = (offset + 2) * " "
+    yaml = YAML()
+    yaml.preserve_quotes = True
+    yaml.width = 4096
+    stream = io.StringIO()
+    yaml.dump(spec_inputs, stream)
     formated_inputs = f"\n{"\n".join(
         f"{indent}{line}"
-        for line in yaml.dump(spec_inputs).splitlines()
+        for line in stream.getvalue().splitlines()
     )}\n\n"
```

Comment:
I _think_ one of the benefits of ruamel is that it can handle this type of manipulation for you. Might be worth seeing if this can be better written to add the inputs to the inputs key.